app/api/v2/views/authentication.py

```diff
@@ -37,70 +34,67 @@ def post(self):
         password = generate_password_hash(data['password'])
 
         try:
-            access_token = create_access_token(identity=data['username']),
-            refresh_token = create_refresh_token(identity=data['username'])    
             user = self.db.get_username_user(username)
             if user:
-                return make_response(jsonify({
-                    "message": "User {} already exists".format(data['username'])
-                }))
+                return make_response(
+                    jsonify({
+                        "message":
+                        "User {} already exists".format(data['username'])
+                    }), 404)
+            
             sign_up = self.db.save(firstname, lastname, othernames, username,
-                                email, phonenumber, password)
+                                   email, phonenumber, password)
 
             return make_response(jsonify({
                 "status": 201,
-                "data": [{
-                    "access_token": access_token,
-                    "refresh_token": refresh_token,
-                    "incident_created": sign_up,
-                    "message": "Created {} successfuly, you can now login ".format(username)
-                }]
+                "message":
+                "Created {} successfuly, you can now login ".format(username),
             }), 201)
         except:
             return make_response(jsonify({
```

Comment:
chose one to use, avoid using both make_response and jsonify at the same time

Reply:
Thank you, I'll implement this.